src/main/scripts/was-check.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+#      Copyright (c) Microsoft Corporation.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+# 
+#           http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Get tWAS installation properties
+source /datadrive/virtualimage.properties
+
+echo "Checking at + $(date)" > /var/log/cloud-init-was.log
+
+# Read custom data from ovf-env.xml
+customData=`xmllint --xpath "//*[local-name()='Environment']/*[local-name()='ProvisioningSection']/*[local-name()='LinuxProvisioningConfigurationSet']/*[local-name()='CustomData']/text()" /var/lib/waagent/ovf-env.xml`
+IFS=',' read -r -a ibmIdCredentials <<< "$(echo $customData | base64 -d)"
+
+# Check whether IBMid is entitled or not
+entitled=false
+if [ ${#ibmIdCredentials[@]} -eq 2 ]; then
+    userName=${ibmIdCredentials[0]}
+    password=${ibmIdCredentials[1]}
+    
+    ${IM_INSTALL_DIRECTORY}/eclipse/tools/imutilsc saveCredential -secureStorageFile storage_file \
+        -userName "$userName" -userPassword "$password" -passportAdvantage
+    if [ $? -eq 0 ]; then
+        output=$(${IM_INSTALL_DIRECTORY}/eclipse/tools/imcl listAvailablePackages -cPA -secureStorageFile storage_file)
+        echo $output | grep -q "ND.v90_9.0.5007" && entitled=true
```

Comment:
We should not hardcode `ND.v90_9.0.5007` here. If we change `WAS_ND_TRADITIONAL=com.ibm.websphere.ND.v90_9.0.5007.20210301_1241` in virtualimage.properties to 9.0.5.8 then this code will break.

Reply:
Although I think `ND.v90_9.0.5007` should be listed anyway as the output of `${IM_INSTALL_DIRECTORY}/eclipse/tools/imcl listAvailablePackages` has nothing to do with the version of tWAS to be installed, I agree with you that adding `ND.v90_9.0.5007` to `virtualimage.properties` is better than hard-coding it.

Reply:
Fixed in commit [2c7ee2f](https://github.com/WASdev/azure.websphere-traditional.image/commit/2c7ee2fb76d6637fbf6fb9e13be44697ab886519#diff-61dca8c2ca0f4d5e8395fef4babb452980cb508cca5b937c110f103065cb2faaR36).